.vscode/launch.json

```diff
@@ -4,6 +4,17 @@
     // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
     "version": "0.2.0",
     "configurations": [
+        {
+            "name": "YAML Linter",
+            "type": "coreclr",
+            "request": "launch",
+            "preLaunchTask": "build-yaml-linter",
+            "program": "${workspaceFolder}/bin/Content.YAMLLinter/Content.YAMLLinter.dll",
+            "cwd": "${workspaceFolder}/Content.YAMLLinter",
+            "console": "internalConsole",
+            "stopAtEntry": false
```

Comment:
Wizden has a firm "don't automatically lint things" policy to prevent whitespace changes cluttering PRs and playing havoc on downstreams.

Reply:
Well this yaml linter is already a required test, so I see no reason why not to include this. It's a good pre-pr check to make sure you didn't introduce anything that would make your PR fail. 

I think you're confusing yamllinter with a yaml formatter.

Reply:
This isn't a change because of a linter, this is a change because I moved this to the top.

Reply:
All I did with this YAML linter run is move it to the top of the run & debug menu, I didn't add this, I didn't change this, nothing